packages/runtime/src/helpers/files.ts

```diff
@@ -74,32 +73,28 @@ export const getMiddleware = async (publish: string): Promise<Array<string>> =>
 
 // eslint-disable-next-line max-lines-per-function
 export const moveStaticPages = async ({
-  netlifyConfig,
-  target,
+  distDir,
   i18n,
   basePath,
+  publishDir,
 }: {
-  netlifyConfig: NetlifyConfig
-  target: 'server' | 'serverless' | 'experimental-serverless-trace'
+  distDir: string
   i18n: NextConfig['i18n']
   basePath?: string
+  publishDir
 }): Promise<void> => {
   console.log('Moving static page files to serve from CDN...')
-  const outputDir = join(netlifyConfig.build.publish, target === 'server' ? 'server' : 'serverless')
+  const outputDir = join(distDir, 'server')
```

Comment:
Why have we removed serverless from here?

Reply:
We force builds to use `server` target anyway, and in Next 13 `serverless` breaks the build.